src/AutoSubscriptions.ts

```diff
@@ -295,9 +302,9 @@ function makeAutoSubscribeDecorator(shallow = false, defaultKeyValues: string[])
     };
 }
 
-export const autoSubscribe = makeAutoSubscribeDecorator(true, [StoreBase.Key_All]);
+export const autoSubscribe = makeAutoSubscribeDecorator(true, undefined);
 export function autoSubscribeWithKey(keyOrKeys: KeyOrKeys) {
-    assert(keyOrKeys || _.isNumber(keyOrKeys), 'Must specify a key when using autoSubscribeWithKey');
+    assert(keyOrKeys !== undefined, 'Must specify a key when using autoSubscribeWithKey');
```

Comment:
I don't think you need this change. It was allowing array, string, and number except zero or empty string.